code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -359,4 +359,387 @@ Difficulty: Very Hard
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	luminosity = 4
+	faction = list("neutral")
+	languages_spoken = 0
+	languages_understood = ALL
+	del_on_death = 1
+	minbodytemp = 0
+	maxbodytemp = 1500
+	unsuitable_atmos_damage = 0
+	flying = 1
+	var/heal_power = 5
+
+/mob/living/simple_animal/hostile/lightgeist/New()
+	..()
+	verbs -= /mob/living/verb/pulled
+	verbs -= /mob/verb/me_verb
+
+/mob/living/simple_animal/hostile/lightgeist/AttackingTarget()
+	..()
+	if(isliving(target) && target != src)
+		var/mob/living/L = target
+		if(L.stat < DEAD)
+			L.heal_overall_damage(heal_power, heal_power, 1)
+			PoolOrNew(/obj/effect/overlay/temp/heal, list(get_turf(target), "#80F5FF"))
+
+/mob/living/simple_animal/hostile/lightgeist/Life()
+	if(!ckey)
+		death()
+	..()
+
+
+/obj/machinery/anomalous_crystal/refresher //Deletes and recreates a copy of the item, "refreshing" it. Only works once per item.
+	activation_method = "touch"
+	cooldown_add = 50
+
+/obj/machinery/anomalous_crystal/refresher/ActivationReaction(mob/user, method)
+	if(..())
+		var/list/L
+		var/turf/T = get_step(src, dir)
+		for(var/i in T)
+			if(istype(i, /obj/item) && !istype(i, /obj/item/weapon/storage) && !(i in affected_targets) && !istype(i, /obj/item/weapon/implant) && !istype(i, /obj/item/weapon/implanter) && !istype(i, /obj/item/weapon/disk/nuclear))
+				var/obj/item/W = i
+				if(!W.admin_spawned)
+					L += W
+		var/obj/item/CHOSEN = pick(L)
+		if(CHOSEN)
+			var/A = new CHOSEN.type(T)
+			affected_targets += A
+			qdel(CHOSEN)
+
+/obj/machinery/anomalous_crystal/possessor //Allows you to bodyjack small animals, then exit them at your leisure, but you can only do this once per activation. Because they blow up. Also, if the bodyjacked animal dies, SO DO YOU.
+	activation_method = "touch"
+
+/obj/machinery/anomalous_crystal/possessor/ActivationReaction(mob/user, method)
+	if(..())
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			var/mobcheck = 0
+			for(var/mob/living/simple_animal/A in range(1, src))
+				if(A.melee_damage_upper > 5 || A.mob_size >= MOB_SIZE_LARGE || A.ckey || A.stat)
+					break
+				A.verbs -= /mob/living/verb/pulled
+				var/obj/structure/closet/stasis/S = new /obj/structure/closet/stasis(A)
+				H.loc = S
+				H.disabilities |= MUTE
+				H.status_flags |= GODMODE
+				H.mind.transfer_to(A)
+				var/obj/effect/proc_holder/spell/targeted/exit_possession/P = new /obj/effect/proc_holder/spell/targeted/exit_possession
+				A.mind.AddSpell(P)
+				mobcheck = 1
+				break
+			if(!mobcheck)
+				new /mob/living/simple_animal/cockroach(loc) //Just in case there aren't any animals on the station, this will leave you with a terrible option to possess if you feel like it
+
+/obj/structure/closet/stasis //Abstract game mechanic workarounds are the best
```

Comment:
If yall you want to do is keep them alive and frozen in place set notransform and godmode on the mob and you're done.